server/src/test/java/io/crate/expression/scalar/ArrayMaxFunctionTest.java

```diff
@@ -0,0 +1,67 @@
+package io.crate.expression.scalar;
+
+import io.crate.expression.symbol.Literal;
+import io.crate.testing.TestingHelpers;
+import io.crate.types.ArrayType;
+import io.crate.types.DataType;
+import io.crate.types.DataTypes;
+import org.junit.Test;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import static io.crate.testing.Asserts.assertThrows;
+
+public class ArrayMaxFunctionTest extends ScalarTestCase {
+
+    @Test
+    public void test_array_returns_max_element() {
+        List<DataType> typesToTest = new ArrayList(DataTypes.PRIMITIVE_TYPES);
+        typesToTest.add(DataTypes.NUMERIC);
+
+        for(DataType type: typesToTest) {
+            var valuesToTest = TestingHelpers.getRandomsOfType(2, 10, type);
+
+            var expected = valuesToTest.stream()
+                .filter(o -> o != null)
+                .max((o1, o2) -> type.compare(o1, o2))
+                .get();
+
+            String expression = String.format("array_max(?::%s[])", type.getName());
```

Comment:
```suggestion
            String expression = String.format(Locale.ENGLISH, "array_max(?::%s[])", type.getName());
```

We randomize the locales in the tests - I don't think it is an issue in this case, but it's a good habit to declare the locale explicitly.
Same in the ArrayMinFunctionTest

(I thought our forbidden-apis check would even complain about this)